Open Judge System/Workers/OJS.Workers.ExecutionStrategies/DotNetCoreProjectTestsExecutionStrategy.cs

```diff
@@ -1,106 +1,149 @@
-        //         return;
-        //     }
-           
-        //     XmlNode generateProgramFileNode = document.CreateNode(XmlNodeType.Element, GenerateProgramFileNodeName, string.Empty);
-        //     XmlNode propertyGroupNode = document.CreateNode(XmlNodeType.Element, PropertyGroupNodeName, string.Empty);
-           
-        //     generateProgramFileNode.InnerText = "false";
-        //     propertyGroupNode.AppendChild(generateProgramFileNode);
-        //     rootNode.AppendChild(propertyGroupNode);
-        //     document.Save(savePath);
-        // }
-           
-        // private void AddPackageReferenceToCsProj(XmlDocument document, string xpathExpression, string savePath, params string[] nodeValues)
-        // {
-        //     XmlNode rootNode = document.DocumentElement;
-        //     if (this.NodeExists(rootNode, xpathExpression))
-        //     {
-        //         return;
-        //     }
-           
-        //     XmlNode packageReference = document.CreateNode(XmlNodeType.Element, PackageReferenceNodeName, string.Empty);
-        //     XmlAttribute includeAttribute = document.CreateAttribute("Include");
-        //     XmlAttribute versionAttribute = document.CreateAttribute("Version");
-        //     includeAttribute.InnerText = nodeValues[0];
-        //     versionAttribute.InnerText = nodeValues[1];
-           
-        //     packageReference.Attributes.Append(includeAttribute);
-        //     packageReference.Attributes.Append(versionAttribute);
-           
-        //     var itemGroupNode = rootNode.SelectSingleNode(ItemGroupNodeXPath);
-        //     itemGroupNode.AppendChild(packageReference);
-        //     document.Save(savePath);
-        // }
-           
-        // private bool NodeExists(XmlNode rootNode, string xpathExpression)
-        // {
-        //     if (rootNode == null)
-        //     {
-        //         throw new ArgumentException("Project file is malformed");
-        //     }
-           
-        //     var targetNode = rootNode.SelectSingleNode(xpathExpression);
-           
-        //     if (targetNode != null)
-        //     {
-        //         return true;
-        //     }
-           
-        //     return false;
-        // }
+        private new const string AdditionalExecutionArguments = "--noresult";
+        private const string CsProjFileExtention = ".csproj";
+        private const string ProjectPathPlaceholder = "##projectPath##";
+        private const string ProjectReferencesPlaceholder = "##ProjectReferences##";
+        private const string NUnitLiteConsoleAppFolderName = "NUnitLiteConsoleApp";
+        private const string UserSubmissionFolderName = "UserProject";
+        private const string NUnitLiteConsoleAppProgramName = "Program";
+        private const string NUnitLiteConsoleAppProgramTemplate = @"
+            using System;
+            using System.Reflection;
+            using NUnit.Common;
+            using NUnitLite;
```

Comment:
New line after `using` block (even in a template :P)

Reply:
Fixed